vars/xwikiBuild.groovy

```diff
@@ -229,6 +229,16 @@ void call(name = 'Default', body)
     }
 }
 
+private def computeDaysToKeepStr(config)
+{
+    def daysToKeepStr = config.daysToKeepStr
+    if (!daysToKeepStr) {
+        daysToKeepStr = '7'
+    }
```

Comment:
You could abbreviate this with:

`return config.daysToKeepStr ?: '7'`

Reply:
Right, I used to have more code in there and forgot to optimize it :)